client.py

```diff
@@ -24,12 +24,16 @@
 if my_map.get("key") == "value":
     print("Successfully connected!")
     print("Now, 'map' will be filled with random entries.")
+
+    iteration_counter = 0
     while True:
         random_key = random.randint(1, 100000)
         random_key_str = str(random_key)
         my_map.put("key" + random_key_str, "value" + random_key_str)
         my_map.get("key" + random_key_str)
-        if random_key % 10 == 0:
+        iteration_counter += 1
```

Comment:
Since we don't use the `random_key` to print the map size anymore, we can change the code above to
```python
random_key = str(random.randint(1, 100000))        
my_map.put("key" + random_key, "value" + random_key)        
my_map.get("key" + random_key)
```
to simplify it a little bit. The same applies to other file

Reply:
That makes sense, thanks. Updated the PR.